### ISSUE TITLE
Freqtrade not completing and displaying backtest results (nothing in logfile)

### ISSUE THREAD

**Issue by Purrbox** [iterativv/NostalgiaForInfinity#364]
I am trying to run a backtest using the default strategy and K-line data that I've previously downloaded. 

It looks like it's working, but then just quits out and ends up back at the command prompt. There is nothing in the logs.

`
ftuser@4969de4f2d75:/workspaces/ft_userdata$ freqtrade backtesting -s SampleStrategy --timerange 20230101-20240101 -p BTC/USDT --timeframe 5m --logfile logs.log
2024-01-10 16:46:06,838 - freqtrade - INFO - freqtrade 2023.12-dev
2024-01-10 16:46:06,855 - freqtrade.configuration.load_config - INFO - Using config: user_data/config.json ...
2024-01-10 16:46:06,861 - freqtrade.loggers - INFO - Verbosity set to 0
2024-01-10 16:46:06,863 - freqtrade.configuration.configuration - INFO - Parameter -i/--timeframe detected ... Using timeframe: 5m ...
2024-01-10 16:46:06,864 - freqtrade.configuration.configuration - INFO - Using max_open_trades: 10 ...
2024-01-10 16:46:06,865 - freqtrade.configuration.configuration - INFO - Parameter --timerange detected: 20230101-20240101 ...
2024-01-10 16:46:06,962 - freqtrade.configuration.configuration - INFO - Using user-data directory: /workspaces/ft_userdata/user_data ...
2024-01-10 16:46:06,966 - freqtrade.configuration.configuration - INFO - Using data directory: /workspaces/ft_userdata/user_data/data/kucoin ...
2024-01-10 16:46:07,640 - freqtrade.configuration.configuration - INFO - Overriding timeframe with Command line argument
2024-01-10 16:46:07,641 - freqtrade.configuration.configuration - INFO - Parameter --cache=day detected ...
2024-01-10 16:46:07,642 - freqtrade.configuration.configuration - INFO - Using pairs ['BTC/USDT']
2024-01-10 16:46:07,643 - freqtrade.configuration.configuration - INFO - Filter trades by timerange: 20230101-20240101
2024-01-10 16:46:07,645 - freqtrade.exchange.check_exchange - INFO - Checking exchange...
2024-01-10 16:46:07,653 - freqtrade.exchange.check_exchange - WARNING - Exchange "kucoin" is known to the the ccxt library, available for the bot, but not officially supported by the Freqtrade development team. It may work flawlessly (please report back) or have serious issues. Use it at your own discretion.
2024-01-10 16:46:07,654 - freqtrade.configuration.config_validation - INFO - Validating configuration ...
2024-01-10 16:46:07,659 - freqtrade.commands.optimize_commands - INFO - Starting freqtrade in Backtesting mode
2024-01-10 16:46:07,660 - freqtrade.exchange.exchange - INFO - Instance is running with dry_run enabled
2024-01-10 16:46:07,661 - freqtrade.exchange.exchange - INFO - Using CCXT 4.1.84
2024-01-10 16:46:07,674 - freqtrade.exchange.exchange - INFO - Using Exchange "KuCoin"
2024-01-10 16:46:12,487 - freqtrade.resolvers.exchange_resolver - INFO - Using resolved exchange 'Kucoin'...
ftuser@4969de4f2d75:/workspaces/ft_userdata$ `

`2024-01-10 16:46:06,861 - freqtrade.loggers - INFO - Verbosity set to 0
2024-01-10 16:46:06,863 - freqtrade.configuration.configuration - INFO - Parameter -i/--timeframe detected ... Using timeframe: 5m ...
2024-01-10 16:46:06,864 - freqtrade.configuration.configuration - INFO - Using max_open_trades: 10 ...
2024-01-10 16:46:06,865 - freqtrade.configuration.configuration - INFO - Parameter --timerange detected: 20230101-20240101 ...
2024-01-10 16:46:06,962 - freqtrade.configuration.configuration - INFO - Using user-data directory: /workspaces/ft_userdata/user_data ...
2024-01-10 16:46:06,966 - freqtrade.configuration.configuration - INFO - Using data directory: /workspaces/ft_userdata/user_data/data/kucoin ...
2024-01-10 16:46:07,640 - freqtrade.configuration.configuration - INFO - Overriding timeframe with Command line argument
2024-01-10 16:46:07,641 - freqtrade.configuration.configuration - INFO - Parameter --cache=day detected ...
2024-01-10 16:46:07,642 - freqtrade.configuration.configuration - INFO - Using pairs ['BTC/USDT']
2024-01-10 16:46:07,643 - freqtrade.configuration.configuration - INFO - Filter trades by timerange: 20230101-20240101
2024-01-10 16:46:07,645 - freqtrade.exchange.check_exchange - INFO - Checking exchange...
2024-01-10 16:46:07,653 - freqtrade.exchange.check_exchange - WARNING - Exchange "kucoin" is known to the the ccxt library, available for the bot, but not officially supported by the Freqtrade development team. It may work flawlessly (please report back) or have serious issues. Use it at your own discretion.
2024-01-10 16:46:07,654 - freqtrade.configuration.config_validation - INFO - Validating configuration ...
2024-01-10 16:46:07,659 - freqtrade.commands.optimize_commands - INFO - Starting freqtrade in Backtesting mode
2024-01-10 16:46:07,660 - freqtrade.exchange.exchange - INFO - Instance is running with dry_run enabled
2024-01-10 16:46:07,661 - freqtrade.exchange.exchange - INFO - Using CCXT 4.1.84
2024-01-10 16:46:07,674 - freqtrade.exchange.exchange - INFO - Using Exchange "KuCoin"
2024-01-10 16:46:12,487 - freqtrade.resolvers.exchange_resolver - INFO - Using resolved exchange 'Kucoin'...
`

**Comment by iterativv** [comment]
Yes, first only BTC pairs it's unlikely to get any trades ... try to set a list with 10-20 pairs at least, then download the data and backtest.
